src/bindings/channel_header.ts

```diff
@@ -1,20 +1,20 @@
 import {AppBinding, AppContext} from 'mattermost-redux/types/apps';
 import {AppExpandLevels} from 'mattermost-redux/constants/apps';
 
-import {getStaticURL, Routes, newChannelHeaderBindings} from '../utils';
+import {getStaticURL, Routes, newChannelHeaderBindings, isConfigured, isConnected, isUserSystemAdmin} from '../utils';
 import {CommandLocations, ZendeskIcon} from '../utils/constants';
 
 // getChannelHeaderBindings returns the users command bindings
-export const getChannelHeaderBindings = (context: AppContext, configured: boolean, connected: boolean, sysadmin: boolean): AppBinding => {
+export const getChannelHeaderBindings = (context: AppContext): AppBinding => {
     const bindings: AppBinding[] = [];
+    const isSysadmin = isUserSystemAdmin(context);
 
     // only show configuration option if admin has not configured the plugin
-    if (!configured && sysadmin) {
+    if (!isConfigured(context) && isSysadmin) {
         bindings.push(channelHeaderConfig(context));
         return newChannelHeaderBindings(bindings);
```

Comment:
could use an `else if` and remove this `return new` statement (and reuse the final one)

Reply:
done.